src/sysupdate/sysupdate-transfer.c

```diff
@@ -603,6 +604,14 @@ int transfer_read_definition(Transfer *t, const char *path, const char **dirs, H
                 return log_syntax(NULL, LOG_ERR, path, 1, SYNTHETIC_ERRNO(EINVAL),
                                   "Source specification lacks Path=.");
 
+        if (!t->source.manifest)
+                t->source.manifest = strdup("SHA256SUMS");
```

Comment:
oom check missing

Reply:
also, i think i would just leave this unset here, and instead fill in SHA256SUMS automatically on use.